packages/adaptive/README.md

```diff
@@ -0,0 +1,158 @@
+# `@gitbook/adaptive`
+
+GitBook supports integrations with popular feature flag providers like [LaunchDarkly](https://launchdarkly.com) and [Bucket](https://bucket.co), making it easy to personalize your documentation based on active feature flags.
+
+This guide walks through how to install and configure each integration using the `@gitbook/adaptive` helper package.
+
+---
+
+## Prerequisites
+
+Make sure you have:
+
+* A GitBook site with adaptive content enabled
+* An active project on LaunchDarkly or Bucket
```

Comment:
I think we should only talk about LD and omit Bucket as we do in the UI:
https://gitbook.slack.com/archives/C08J001JLCD/p1750286067406389

Reply:
Do you think we should remove Bucket references in general? I think in this readme / docs it’s okay to have - but agree in-product we should focus on launchdarkly. But don’t think we should completely hide information for people who might want to use it